README.md

```diff
@@ -11,3 +12,19 @@ This project aims to deal with modules related to the webclient of Odoo. You'll
 - Add widgets
 - Ease the import/export features
 - Generally add clientside functionality
+
+[//]: # (addons)
+This part will be replaced when running the oca-gen-addons-table script from OCA/maintainer-tools.
+[//]: # (end addons)
```

Comment:
I wonder if this part should be add on 8.0 series only or on every branches, does the ``gen_addons_table`` script running on all branches?